src/sass/ResultActionMenu.scss

```diff
@@ -0,0 +1,152 @@
+@import './Variables.scss';
```

Comment:
Why is that suddenly included? It seems out of scope

Reply:
Because the result action css was broken.

Reply:
That's seems like another issue then, please split :D

Reply:
No that css is require for Copy Paste.

Reply:
Explain please, I don't understand why a new ResultAction requires changes in the menu, changes in the QuickviewAction etc.
I'm still not convinced that this is in the scope.

Reply:
Because of the existing css result actions look out of place when compared with quickview result actions.

Reply:
And that's another issue :)

Reply:
So you want me to ship an half feature instead of a complete one ?

Reply:
I want to ship the strict minimum for the feature, this doesn't seems like it.
Can you indicate in this file which CSS rule concerns directly and exclusively the CopyAndPaste component?

Reply:
I agree with jeremie